### ISSUE TITLE
get users' comments from Gdrive files

### ISSUE THREAD

**Issue by abubelinha** [iterative/PyDrive2#182]
Looks like it is possible now to get comments from different parts of Google drive files.
https://developers.google.com/drive/api/v3/reference/comments/list

Is it possible to get them using PyDrive2 also?

If not, would it be possible to implement?

Thanks

**Comment by abubelinha** [comment]
Related question:
Sorry if this is a nonsense but I just can't understand Google oauth2 authentication workflow (too complicated for me).

I think that, once authenticated, we can make [Google API calls using the Python requests library](https://google-auth.readthedocs.io/en/latest/user-guide.html#requests). 
So in theory users comments can be grabbed using this code (taken from [here](https://github.com/burnash/gspread/issues/978#issuecomment-1155426260)):

##### code 2:
```Python
url = "https://www.googleapis.com/drive/v3/files/{}/comments".format(file_id)
params = {
    "fields": "*",
}
res = session.get(url, params=params)
response = res.json()
```

As I am going to login Gdrive using Pydrive2 anyway, I wonder if it is possible to get that `session` object from PyDrive2 easy oauth2 authentication using my credentials file?  So I can run the above code 2 after having running first this code:
##### code 1:
```Python
from pydrive2.auth import GoogleAuth
from pydrive2.drive import GoogleDrive
gauth = GoogleAuth()
# Try to load saved client credentials
gauth.LoadCredentialsFile(credsFile)
drive = GoogleDrive(gauth)
```

In summary, how to go from `gauth` to `session`?

Thanks

**Comment by abubelinha** [comment]
Closing issue.
I found some [related code samples](https://github.com/burnash/gspread/issues/978#issuecomment-1156475616) in gspread library.
